cli-tests/tests/pmm-cli/pmm.spec.ts

```diff
@@ -3,6 +3,10 @@ import * as cli from '@helpers/cliHelper';
 import PMMRestClient from '@support/types/request';
 
 test.describe('PMM binary tests @pmm-cli', async () => {
+  test.afterEach(async ({}) => {
+    await cli.exec('docker rm -f $(docker ps -a -q)')
```

Comment:
does this remove all docker containers? I think we should remove only ones that we started

Reply:
I just learned the hard way. All my containers are gone :)